emmet-core/emmet/core/openff/benchmarking.py

```diff
@@ -0,0 +1,99 @@
+from pydantic import BaseModel, Field
+from typing import Optional
+
+import warnings
+
+from MDAnalysis import Universe
+from MDAnalysis.analysis.dielectric import DielectricConstant
+
+from transport_analysis.viscosity import ViscosityHelfand
+
+
+class SolventBenchmarkingDoc(BaseModel, arbitrary_types_allowed=True):
+    density: Optional[float] = Field(None, description="Density of the solvent")
+
+    viscosity_function_values: Optional[list[float]] = Field(
+        None, description="Viscosity function over time"
+    )
+
+    viscosity: Optional[float] = Field(None, description="Viscosity of the solvent")
+
+    dielectric: Optional[float] = Field(
+        None, description="Dielectric constant of the solvent"
+    )
+
+    job_uuid: Optional[str] = Field(
+        None, description="The UUID of the flow that generated this data."
+    )
+
+    flow_uuid: Optional[str] = Field(
+        None, description="The UUID of the top level host from that job."
+    )
+
+    dielectric_run_kwargs: Optional[dict] = Field(
+        None, description="kwargs passed to the DielectricConstant.run method"
+    )
+
+    viscosity_run_kwargs: Optional[dict] = Field(
+        None, description="kwargs passed to the ViscosityHelfand.run method"
+    )
+
+    tags: Optional[list[str]] = Field(
+        [], title="tag", description="Metadata tagged to the parent job."
+    )
+
+    @classmethod
+    def from_universe(
+        cls,
+        u: Universe,
+        temperature: Optional[float] = None,
+        density: Optional[float] = None,
+        job_uuid: Optional[str] = None,
+        flow_uuid: Optional[str] = None,
+        dielectric_run_kwargs: Optional[dict] = None,
+        viscosity_run_kwargs: Optional[dict] = None,
+        tags: Optional[list[str]] = None,
+    ) -> "SolventBenchmarkingDoc":
+        if temperature is not None:
+            dielectric = DielectricConstant(
+                u.atoms, temperature=temperature, make_whole=False
+            )
+            dielectric_run_kwargs = dielectric_run_kwargs or {}
+            dielectric.run(**dielectric_run_kwargs)
+            eps = dielectric.results.eps_mean
+        else:
+            warnings.warn(
```

Comment:
for these warnings, when we're going to run the builders in a distributed cluster how are we meant to suppress these?

Just with this on the builder side?
```
with warnings.catch_warnings():
    warnings.simplefilter("ignore")
    SolventBenchmarkingDoc.from_universe(...)
    etc.
```

Reply:
same thought on the other `warnings.warn`s

Reply:
Very good point, this isn't something I was considering. I am happy to implement whatever solution is best/standard. If that's the solution you described above I can add it in now.

Reply:
we have the filter warnings function in other builders, so I would deem that as okay if having easy access to these warnings at the document model is important to you for testing, etc.

Could also probably do something will the logger for the builder to move these messages over to the builder side, but that isn't entirely necessary at this point imo

Reply:
There are some warning's in PR #1088 that I'll do that for but I'll just ax this one. It's not really user facing so the temperature should always be available.